cache/in-memory/src/model/guild.rs

```diff
@@ -7,40 +7,99 @@ use twilight_model::{
     id::{ApplicationId, ChannelId, GuildId, UserId},
 };
 
+/// Represents a cached [`Guild`].
+///
+/// [`Guild`]: twilight_model::guild::Guild
 #[derive(Clone, Debug, Eq, PartialEq, Serialize)]
 pub struct CachedGuild {
-    pub id: GuildId,
+    /// ID of the AFK channel.
     pub afk_channel_id: Option<ChannelId>,
+    /// AFK timeout in seconds.
     pub afk_timeout: u64,
+    /// For bot created guilds, the ID of the creating application.
     pub application_id: Option<ApplicationId>,
+    /// Banner hash.
+    ///
+    /// See [Discord Docs/Image Formatting].
+    ///
+    /// [Discord Docs/Image Formatting]: https://discord.com/developers/docs/reference#image-formatting
     pub banner: Option<String>,
+    /// Default message notification level.
     pub default_message_notifications: DefaultMessageNotificationLevel,
+    /// For Community guilds, the description.
     pub description: Option<String>,
+    /// For discoverable guilds, the discovery splash hash.
+    ///
+    /// See [Discord Docs/Image Formatting].
+    ///
+    /// [Discord Docs/Image Formatting]: https://discord.com/developers/docs/reference#image-formatting
     pub discovery_splash: Option<String>,
+    /// Explicit content filter level.
     pub explicit_content_filter: ExplicitContentFilter,
+    /// Enabled [guild features].
+    ///
+    /// [guild features]: https://discord.com/developers/docs/resources/guild#guild-object-guild-features
     pub features: Vec<String>,
+    /// Icon hash.
+    ///
+    /// See [Discord Docs/Image Formatting].
+    ///
+    /// [Discord Docs/Image Formatting]: https://discord.com/developers/docs/reference#image-formatting
     pub icon: Option<String>,
+    /// ID of the guild.
+    pub id: GuildId,
+    /// ISO8601 timestamp of the user's join date.
```

Comment:
ISO 8601 is two words

Reply:
I copied this from discord but i'll fix it anyway